Add money_only_cents helper method

carlomartinucci opened the pull request:
Following issue #397, I think this helper method may be useful.

I'm sorry if I break some convention or standard.

antstorm commented on lib/money-rails/helpers/action_view_extension.rb:
```diff
@@ -48,5 +48,16 @@ def money_without_cents(value, options={})
     end
+
+    def money_only_cents(value)
+      if value.is_a?(Money)
```
There's probably no need for this empty branch

carlomartinucci replied:
thank you, i didn't realize that `money.to_money == money`.

carlomartinucci commented:
@antstorm thank you, I added the spec (hope it's good, I followed the other ones).
My specific use case was for styling reason, so `.format(html_wrap: true)` and a little more css would do.

Another use case could be, for instance, if you want integer part and cents part in different td of the same row in a table, or similar complex settings.
Another hypothetical use case could be to hide the real amounts in a list but make them distinguishable, similar to how credit card numbers work.

antstorm commented:
@carlomartinucci the spec looks good, but can you also verify the exception case when a non-money object was given. It might also make sense to check that a single digit is prefixed with zero, just in case.

carlomartinucci commented:
@antstorm I hope I understood. I added test for a monetizable object (a number) and for less than 10 cents. Do you want me to add also a test for an object that does not respond to `to_money`? (e.g. `true`?)

antstorm commented on spec/helpers/action_view_extension_spec.rb:
```diff
@@ -74,6 +74,31 @@
+    it { is_expected.to be_a String }
+    it { is_expected.not_to include Money.default_currency.decimal_mark }
+    it { is_expected.not_to include Money.default_currency.symbol }
+    it { is_expected.to include "00" }
```
This should be `is_expected.to eq "00"` (instead of `include`), otherwise result of `12500` would also pass this spec

carlomartinucci replied:
good catch, thanks

antstorm commented:
@carlomartinucci yeah, that's exactly right, a spec case that takes `return '00' unless value.respond_to?(:to_money)` path, otherwise all the specs that you have will actually respond to `to_money`

carlomartinucci commented:
@antstorm I added the spec for `false` (the only non-monetizable object i could think of, beside `true` ;) ) and also removed some useless `it`'s, since `is_expected.to eq ...` covers it all. :)

antstorm commented:
I wonder why is that failing on CI…

carlomartinucci commented:
I really have no idea... Should I do anything?
